### ISSUE TITLE
function toggleDoor(door: number)

### ISSUE THREAD

**Issue by LandminenTester** [Stuyk/rebar-altv#152]
The function **toggleDoor** currently just opens a door of a vehicle but it cant close it.

The used function .getDoorState(door) always returns 0

```ts
    /**
     * Toggle door state for the given vehicle
     *
     * 0 - Driver
     * 1 - Passenger
     * 2 - Back Left
     * 3 - Back Right
     * 4 - Hood
     * 5 - Trunk
     *
     * @param {number} door
     */
    function toggleDoor(door: number) {
        vehicle.setDoorState(door, vehicle.getDoorState(door) === 0 ? 7 : 0);
    }
```

**Comment by LandminenTester** [comment]
Currently a altv bug: 

https://github.com/altmp/altv-issues/issues/737

**Comment by Stuyk** [comment]
Should be resolved in #132 with a small override
